docs/cn_doc/cn-README.md

```diff
@@ -0,0 +1,37 @@
+# 欢迎使用Nebula Graph官方文档。
+---
+
+**Nebula Graph** 是一个分布式，可扩展，响应迅速的图数据库,是世界上唯一能够容纳百亿个顶点（节点）和万亿条边的图数据库解决方案，并同时提供毫秒级延迟。
+
+**Nebula Graph** 的目标是为超大规模图数据提供高并发，低延时的读、写及计算。Nebula是一个开源项目，我们期待与社区合作共同推广图数据库。
+
+**Nebula Graph** 的主要特点：
+
+ * 对称分布
+ * 可扩展
+ * 高容错
+ * 类似SQL的查询语言
+
+## Nebula获取 ##
+
+**Nebula Graph** 源码参见[GitHub](https://github.com/vesoft-inc/nebula)。目前的版本号为R1910_alpha。
```

Comment:
you can remove the current version sentence since we won't change the doc every time